pkg/docker/docker_api.go

```diff
@@ -0,0 +1,283 @@
+package docker_api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/appscode/analytics/pkg/spreadsheet"
+	"github.com/appscode/errors"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/sheets/v4"
+)
+
+type DockerRepositories struct {
+	Count    int         `json:"count"`
+	Next     interface{} `json:"next"`
+	Previous interface{} `json:"previous"`
```

Comment:
`Next` and `Previous` should *string
As
```json
{
  "count": 93,
  "next": "https://hub.docker.com/v2/repositories/appscode/?page=3&page_size=5",
  "previous": "https://hub.docker.com/v2/repositories/appscode/?page=1&page_size=5"
}

// When there is no page available
{
  "count": 93,
  "next": null,
  "previous": null
}
```

we should get data until `Next = nil`